environment.yml

```diff
@@ -1,6 +1,5 @@
 name: raven
 channels:
-- zeitsperre
```

Comment:
https://github.com/CSHS-CWRA/RavenPy/blob/edab9915b70be8e4b0a892be7e7a0e18036049bb/environment.yml#L3

need the same ?

Reply:
As of yesterday, yes. My repo source is obsolete.